### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by mbrg**
Hi SPIFFE team,
These are some glitches I've encountered while reading through the spec documents. These are minor fixes, so it would be reasonable if you choose to close this PR and push the change in some other bigger revision :).

Hope this helps.

### CODE REVIEW

**Review by spikecurtis — DISMISSED** [review]

**Review by justinburke — DISMISSED** [review]

File: standards/SPIFFE-ID.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ An SVID is a fairly simple construct, and comprises three basic components:
 * A valid signature
 * An optional public key
 
-The SPIFFE ID and the public key MUST be included in a portion of the payload which is signed. If a public key is included, then the corresponding private key is retained by the entity to which the SVID has been issued, and is used to prove ownership of the SVID itself.
+The SPIFFE ID and the valid signature MUST be included in a portion of the payload which is signed. If a public key is included, then the corresponding private key is retained by the entity to which the SVID has been issued, and is used to prove ownership of the SVID itself.
```
Comment (evan2645) [inline standards/SPIFFE-ID.md]: Suggestion

`The SPIFFE ID and the public key (if present) MUST` ...

The signature itself is not included in a portion of the payload which is signed

Comment (mbrg) [reply standards/SPIFFE-ID.md]: Thank you, I've implemented the change in 7cf5971.

**Review by evan2645 — APPROVED** [review]
Thank you @mibarg!

**Comment by evan2645** [comment]
Going to go ahead and merge this as the most recent changes were discussed and verbally approved in the last SIG-Spec call.

**Pull request merged**